api/v1alpha1/README.md

```diff
@@ -4,24 +4,46 @@ This document is intended to provide context to the API definition.
-| PIDPressure | True if pressure exists on the processes—that is, if there are too many processes on the node, otherwise False. | 
-| MemoryPressure |True if pressure exists on the device memory—that is, if the device memory is low, otherwise False. |
-| CPUPressure |True if device is experiencing high CPU utilization that might affect the performance of the workloads running on that device, otherwise False. |
-| SystemdUnitsRunning | True is all specified systemd units are running, otherwise False. |
-| ContainersRunning | True is all specified containers are running, otherwise False. |
+| ID | Unique identifier for the application |
+| Name | A human readable name for the application |
+| Status | The current lifecycle phase of an application |
+| Ready  | Is the application ready to accept work |
+| Restarts | The number of times an application has been restarted |
+
+### Device.Status.Application.Status
+
+| Name    | Desc |
+| -------- | ------- |
+| Preparing | The application is being validated and dependencies are being created/pulled |
+| Starting | The application is being started but is not yet available |
+| Running | The application has been successfully scheduled and the process has not existed |
+| Error | Part of all of the application has been terminated with a non zero exit code |
+| Unknown | The Device us unable to determine the exact status |
+| Completed | The application has exited successfully with a zero exit code. |
 
 ### Conditions
-These conditions that cover the operation of the agent
+
+#### General
+These conditions that cover the general operation of the Agent/Device
 
 | Name    | Desc |
 | -------- | ------- |
-| Available  | True if the controller is available for deploying configurations and postAction commands otherwise False  |
-| Progressing | True if the controller is in the process of reconciling Spec, otherwise False    |
-| Degraded    | True if the controllers functionality is impaired in some way but not unavailable, otherwise False |
\ No newline at end of file
+| Available  | True if the device is available for reconciling changes to the Device Spec otherwise False  |
+| Updating | True if the controller is in the process of reconciling Spec, otherwise False    |
+| Degraded    | True if the device has reported a transient retryable error or failure during its operations, otherwise False |
+
+#### Applications/Workloads
+These are the core conditions that cover the ability of the Device to accept/manage applications.
+
+| Name    | Desc |
+| -------- | ------- |
+| AppsAvailable  | True if the Device reports no resource pressure and can accept new application; otherwise is False. |
+| AppsDegraded   | True if one or more of the applications has reported error; otherwise False |
+| DiskPressure | True if pressure exists on the disk size—that is, if the disk capacity is low; otherwise False. |
```

Comment:
The DDD describes "warning" and "critical" (= alarm) watermarks.

Reply:
that would be implementation details but I will update the description.
`DiskPressure=true, Reason=Warning, Message="details of warning"`
`DiskPressure=true, Reason=Alarm, Message="details of alarm"`